gdsfactory/components/coupler_adiabatic.py

```diff
@@ -1,12 +1,42 @@
-from typing import Tuple
-
-import picwriter.components as pc
+from gdspy import Path
+from phidl import Device as Component
```

Comment:
from gdsfactory.component import Component

we added many improvements over phidl.Device